release-notes/1.1.2.md

```diff
@@ -0,0 +1,4 @@
+# 1.1.2
+
+Include message attributes when retrying messages from the DLQ to the main queue.
```

Comment:
So, to confirm the intention, this is a bug fix as previously it lost the attributes when retrying?

Reply:
I am not sure if it was a bug or intentionally omitted. I had a look through our other projects & none of them (except for `dps-data-compliance` ) use custom attributes so the tool is fine for 'hmpps' purposes

Reply:
So my only concern is that it doesn't break existing services that use this library.

But given that you get the attributes from the DLQ and pass them over - that should be fine then?

Reply:
Yes the attributes are copied over from the message on the DLQ so it should have no impact on any services already using the library